src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+class IsoscelesTrapezoid extends Figure {
+    private double base1;
+    private double base2;
+    private double height;
+
+    public IsoscelesTrapezoid(String color, double base1, double base2, double height) {
+        super(color);
+        this.base1 = base1;
+        this.base2 = base2;
+        this.height = height;
+    }
+
+    @Override
+    public double getArea() {
+        return 0.5 * (base1 + base2) * height;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: isosceles trapezoid, area: "
+                + getArea() + " sq. units, base1: "
+                + this.base1 + ", base2: "
+                + this.base2 + ", height: "
+                + this.height + " units, color: "
+                + this.color);
```

Comment:
Consider using `getColor()` instead of directly accessing the `color` field to maintain encapsulation and adhere to OOP principles.